scalatest/src/main/scala/org/scalatest/tools/ColorBar.scala

```diff
@@ -82,11 +82,12 @@ private[scalatest] class ColorBar extends JPanel {
     repaint()
   }
 
-  override def update(g: Graphics): Unit = {
+  override def update(g: Graphics | Null): Unit = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.